packages/nextjs/components/scaffold-eth/Contract/ReadOnlyFunctionForm.tsx

```diff
@@ -59,7 +59,7 @@ export const ReadOnlyFunctionForm = ({ functionFragment, contractAddress }: TRea
       {inputs}
       <div className="flex justify-between gap-2">
         <div className="flex-grow">
-          {result ? (
+          {result != null ? (
```

Comment:
Do we need the ternary?

Or could it just be:

```ts
          {result != null && (
            <span className="block bg-secondary rounded-3xl text-sm px-4 py-1.5">
              <strong>Result</strong>: {displayTxResult(result)}
            </span>
          )}
```

Reply:
or 
```
{result ?? (<span className="block bg-secondary rounded-3xl text-sm px-4 py-1.5">
    <strong>Result</strong>: {displayTxResult(result)}
  </span>
)}
```

Reply:
I'd prefer `result !== null && result !== undefined` so we and new devs don't need to remember what is `==` to `null`. Or add a link to StackOverflow as comment before this line. For example, if I would meet that line, I'd think any falsy value is `==` to  `null`

Reply:
Ye, thanks for this 🙌 fixed at [3fd4b58](https://github.com/scaffold-eth/se-2/pull/164/commits/3fd4b58e7c70737a760f65fd03201f62a17bc5a3). According to me using `&&` won't cause a problem also tested it works fine. 

> Therefore, if the condition is true, the element right after && will appear in the output. If it is false, React will ignore and skip it.

react [docs ^ ](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator), since we are comparing so this will be either false or true and won't cause any issue

Reply:
> I'd prefer result !== null && result !== undefined so we and new devs don't need to remember what is == to null. Or add a link to StackOverflow as comment before this line. For example, if I would meet that line, I'd think any falsy value is == to null

Added a comment for same at [215b851](https://github.com/scaffold-eth/se-2/pull/164/commits/215b8511edf3b64311d1f4f98f88ff8dc22a49bb)

Reply:
> Added a comment for same at [215b851](https://github.com/scaffold-eth/se-2/pull/164/commits/215b8511edf3b64311d1f4f98f88ff8dc22a49bb)

Sorry, I was on my way to edit it also haha. I kind of prefer the explicit option `result !== null && result !== undefined`. Not as pretty. Not a big deal... just a nitpick :D

> According to me using && won't cause a problem also tested it works fine.

Yep, I think this is only tricky when you use template literals (${}), where `&&` would add "false" to the text.